fpdf/fpdf.py

```diff
@@ -2570,6 +2571,44 @@ def skew(self, ax=0, ay=0, x=None, y=None):
             )
             yield
 
+    @check_page
+    @contextmanager
+    def mirror(self, angle, origin=(None, None)):
+        """
+        This method allows to perform a reflection transformation over a given mirror line.
+        It must be used as a context-manager using `with`:
+
+            with mirror(angle="SOUTH", origin=(15,15)):
+                pdf.something()
+
+        The mirror transformation affects all elements which are printed inside the indented
```

Comment:
```suggestion
        The mirror transformation affects all elements which are rendered inside the indented
```
Just a suggestion for a minor change of wording.

Great docstring by the way!